email/johtoselvitys-valmis.mjml

```diff
@@ -74,15 +78,17 @@
                 </mj-text>
                 <mj-text mj-class="basic-txt">
                     How did we do? Please share your opinion <a
-                        href="https://response.questback.com/isa/qbv.dll/bylink?p=x6TIpQ4sCcJt9Dv1T-AuuKa-DssX27rx6er2ohTVVipooPTA8H0u-weNFWY9GsVI0">
+                        href="https://ecv.microsoft.com/zepZlZSdnT">
                     here</a>.
                 </mj-text>
                 <mj-text mj-class="basic-txt">
                     Kind regards,
                 </mj-text>
                 <mj-text mj-class="signature">
-                    Helsinki Urban Environment Resident and Business Services
+                    Helsinki city
```

Comment:
Should this be City of Helsinki? Or Helsinki City with a capital C?

Reply:
@umeetiusbaar The "Urban Environment Resident and Business Services" text can be found on line 90. Line 88 is correct. Same thing can be found e.g. on lines 24-26 and 57-59.